day_7/number.rb

```diff
@@ -0,0 +1,126 @@
+# A class to turn an integer to its english string
+# This should probably be refactored to a static method in the integer and string class
+# Note, this only goes up to quintillion, so numbers larger than quintillion do not work.
+
+module Words
+    NUM_WORDS = {

```

Comment:
Great looking hash, and nice separation of concerns breaking this into multiple classes and a module.